doc/lua_api.txt

```diff
@@ -4076,6 +4076,9 @@ Definition tables
         nametag = "", -- by default empty, for players their name is shown if empty
         nametag_color = <color>, -- sets color of nametag as ColorSpec
         infotext = "", -- by default empty, text to be shown when pointed at object
+        no_static_save = false,
```

Comment:
`static_save = true,`

Reply:
You can do this by checking for the existence of the static_save bool, and defaulting to true if not present

Reply:
Or by checking if it's not false.